nixos/tests/postgresql/postgresql.nix

```diff
@@ -26,6 +26,15 @@ let
     INSERT INTO sth (id) VALUES (1);
     CREATE TABLE xmltest ( doc xml );
     INSERT INTO xmltest (doc) VALUES ('<test>ok</test>'); -- check if libxml2 enabled
+    -- check if hardening gets relaxed
+    CREATE EXTENSION plv8;
+    DO $$
+      let xs = [];
+      for (let i = 0, n = 4000000000; i < n; i++) {
```

Comment:
Also maybe add a comment that this is trying to trigger V8 JIT, to trip up `MemoryDenyWriteExecute`, since we have no guarantee JIT will actually be used.

Reply:
Not really strong on Javascript, feel free to propose something.

Reply:
That being said, the current query triggers the jit and works as a reproducer.